packages/search/tests/integration/components/navi-search-result/report-test.js

```diff
@@ -0,0 +1,34 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+import setupMirage from 'ember-cli-mirage/test-support/setup-mirage';
+import Service from '@ember/service';
+
+module('Integration | Component | navi-search-result-report', function(hooks) {
+  setupRenderingTest(hooks);
+  setupMirage(hooks);
+
+  hooks.beforeEach(async function() {
+    // Load metadata needed for request fragment
+    await this.owner.lookup('service:bard-metadata').loadMetadata();
+    this.service = this.owner.lookup('service:navi-report-search-provider');
+    const store = this.owner.lookup('service:store'),
+      mockAuthor = store.createRecord('user', { id: 'navi_user' });
+    this.owner.register(
+      'service:user',
+      Service.extend({
+        getUser: () => mockAuthor
+      })
+    );
+  });
+
```

Comment:
a few other ideas for things to test (probably integration tests): 
- special characters in the search string (commas, etc.., since it's user input)
- if something weird comes back from the ws, like an empty object comes back without a title.